python/tidy.py

```diff
@@ -613,11 +631,11 @@ def scan(faster=False, progress=True):
     checking_functions = (check_flake8, check_lock, check_webidl_spec, check_json)
     line_checking_functions = (check_license, check_by_line, check_toml, check_rust, check_spec)
     errors = collect_errors_for_files(files_to_check, checking_functions, line_checking_functions)
-
-    # wpt lint checks
-    wpt_lint_errors = check_wpt_lint_errors(get_wpt_files(faster, progress))
-    # collect errors
-    errors = itertools.chain(errors, wpt_lint_errors)
+    if not faster:
```

Comment:
I got confused by the --help message. It says the following:
Command Arguments:
  --faster       Only check changed files and skip the WPT lint in tidy
  --no-progress  Don't show progress for tidy
I read the conversation in #9778 again, may I change the help message?